api/pkg/transformer/jsonpath/jsonpath_test.go

```diff
@@ -20,6 +21,12 @@ const (
 		  "instances": [
 			{"sepal_length":2.8, "sepal_width":1.0, "petal_length":6.8, "petal_width":0.4},
 			{"sepal_length":0.1, "sepal_width":0.5, "petal_length":1.8, "petal_width":2.4}
+		  ],
+		  "empty_array": [],
```

Comment:
Can you add test where the array has some null field
```
"instances_with_null": [
			{"sepal_length":2.8, "sepal_width":1.0, "petal_length":6.8, "petal_width":0.4},
			{"sepal_length":0.1, "sepal_width":0.5, "petal_length":1.8, "petal_width":null},
                         {"sepal_length":0.1, "sepal_width":0.5, "petal_length":1.8, "petal_width":0.5}
		  ],
```

and  json path `$.instances_with_null[*].petal_width`